src/tscpp/api/utils_internal.cc

```diff
@@ -260,7 +260,7 @@ utils::internal::invokePluginForEvent(GlobalPlugin *plugin, TSHttpTxn ats_txn_ha
 }
 
 void
-utils::internal::invokePluginForEvent(GlobalPlugin *plugin, TSHttpAltInfo altinfo_handle, TSEvent event)
+utils::internal::invokePluginForEvent(GlobalPlugin *plugin, TSHttpAltInfo altinfo_handle, [[maybe_unused]] TSEvent event)
```

Comment:
There is an assertion in this function which uses the `event` argument.
That's why it's with `[[maybe_unused]]` instead of like the other ones.